src/test/java/tests/BaseTest.java

```diff
@@ -0,0 +1,34 @@
+package tests;
+
+import org.junit.jupiter.api.*;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.PageFactory;
+import pages.HomePage;
+
+import java.time.Duration;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class BaseTest {
+
+    private WebDriver driver;
+    protected HomePage homePage;
+
+    @BeforeAll
+    public void start () {
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        driver.manage().window().maximize();
+        homePage = PageFactory.initElements(driver, HomePage.class);
```

Comment:
already saw init elements at base page

Reply:
since base page is abstract and base for all derivatives its enough to make this call at constructor